server/config/config.go

```diff
@@ -21,6 +22,8 @@ const (
 	DefaultGRPCWebAddress = "0.0.0.0:9091"
 )
 
+var ErrMinGasPrices = errors.New("please set min gas price in app.toml or flag or env var")
```

Comment:
maybe we can keep it private?

Reply:
I think to remove this variable, cause it is used only once.